### DISCUSSION TITLE
Porting Xamarin libraries to Maui (uwp)

### DISCUSSION THREAD

**Discussion by Pouria7** [dotnet/maui#7836] [General]
As I see Most Xamarin Nuget packages are written for Android, iOS, and UWP...
converting uwp parts to winui3 may not be easy, especially for libraries that do not rely on ui like downloadManager,background service and more.
Most of these libs (projects with a small community) going to lost.
It's take time to redev the libs and maui make usable for all .net developers.
Is there any plan to simplify this process ?

**Comment by xtuzy** [comment]
maui use Winui3, not wpf, xamarin.form's uwp use winui2

**Reply by Pouria7** [reply]
Ok, I change it to winui3, I actually mean win32
